### PR TITLE
Refining some issues in the vector renderer.

### PR DESCRIPTION
**Pull request by bengarney**
Now loads an SVG twice and isn’t broken by live reload.

### CODE REVIEW

**Comment by bengarney** [comment]
@Newtsuck @SmilyOrg Please review/land.

File: loom/graphics/gfxVectorRenderer.cpp [hunk 1]
```diff
@@ -451,35 +454,41 @@ void VectorSVG::resetInfo() {
...
     if (image != NULL) {
         nsvgDelete(image);
         image = NULL;
     }
 }
+
 void VectorSVG::loadFile(utString path, utString units, float dpi) {
 	reset();
-	this->units = utString(path);
+	this->units = units;
```
Comment (SmilyOrg) [inline loom/graphics/gfxVectorRenderer.cpp]: Oops! Weird, probably bad copypaste by me!

File: loom/graphics/gfxVectorRenderer.cpp [hunk 2]
```diff
@@ -451,35 +454,41 @@ void VectorSVG::resetInfo() {
...
 	this->dpi = dpi;
 	this->path = lmNew(NULL) utString(path);
 	reload();
-	loom_asset_subscribe(path.c_str(), onReload, this, false);
+	loom_asset_subscribe(path.c_str(), onReload, this, true);
```
Comment (SmilyOrg) [inline loom/graphics/gfxVectorRenderer.cpp]: Why does it load the SVG twice now?

**Comment by SmilyOrg** [comment]
Just one question!

File: loom/graphics/gfxVectorRenderer.cpp (hunk 2 above)
Comment (bengarney) [inline loom/graphics/gfxVectorRenderer.cpp]: It doesn't, I just changed the flag so onReload is ALWAYS called?

Comment (SmilyOrg) [inline loom/graphics/gfxVectorRenderer.cpp]: Doesn't `reload();` before it take care of that already? I think it should be either that the `reload();` is there or that `loom_asset_subscribe` is `true`, but not both?

Comment (bengarney) [inline loom/graphics/gfxVectorRenderer.cpp]: When I made this change it fixed the live reload issues that Josh reported where any load of an SVG after the first didn't show up. Maybe reload() doesn't work properly in this case?

Comment (SmilyOrg) [inline loom/graphics/gfxVectorRenderer.cpp]: Hmm, interesting! Can you try removing `reload();` and seeing if it still works?

**Comment by bengarney** [comment]
OK trying it now... waiting on build box.

File: loom/graphics/gfxVectorRenderer.cpp (hunk 2 above)
Comment (bengarney) [inline loom/graphics/gfxVectorRenderer.cpp]: Yes I can!

**Comment by bengarney** [comment]
It wasn't extraneous, and further, I do need the copy. WTF. But it seems stable now. @Newtsuck Please review and land at your earliest convenience.

**Comment by Newtsuck** [comment]
Tested and works! Landing! (finally)

**Pull request merged**